src/ui/preferences/GeneralSettings.qml

```diff
@@ -41,6 +41,14 @@ QGCView {
 
     QGCPalette { id: qgcPal }
 
+    FileDialog {
```

Comment:
The QML file dialog is pretty horrendous. We only use it as a last resort on mobile builds. Give me a couple days and I'll make a QGCFileDialog QML control that uses native dialog on desktop and Qml dialog on mobile. I need it for some work I"m doing anyways.